R/LSgameofchance.R

```diff
@@ -147,154 +148,144 @@ compare_function2 <- function(k1, t, p, simulation){
 ##
 LSgameofchance   <- function(jaspResults, dataset, options, state = NULL){
 
-  #ready <- (length(options$variables) > 0)
+  # input values
+  nPlayers  <- length(options[["players"]])
+  probWin   <- sapply(options[["players"]], function(p)p$values[[1]])
+  xPoints   <- sapply(options[["players"]], function(p)p$values[[2]])
+  winPoints <- options[["winPoints"]]
+  nSims     <- options[["nSims"]]
   
+  # normalizing the probability of wining:
+  probWin   <- probWin / sum(probWin) # normalit
```

Comment:
it's better for maintaining the code if the variables are named more intuitively then a single letter (I tried to change the naming consistently in this one main function, it would be great if you could propagate the changes to the remaining functions)